### PR TITLE
Leaf/b1.2_http_post_call_handling

### PR DESCRIPTION
**Pull request by bluesimp1102**
Issue: https://github.com/LetsMesh/Site/issues/67
Original PR: https://github.com/LetsMesh/Site/pull/222
Creating a new PR because the old branch was from main instead of dev_new

### CODE REVIEW

File: mesh/accounts/views.py [hunk 1]
```diff
@@ -1 +1,105 @@
...
+            phoneNum = data['phoneNum'], 
+            displayTheme = data['displayTheme'], 
+            enabled2Factor = data['enabled2Factor'],
+            isMentor = data['isMentor'],
+            isMentee = data['isMentee']
+        )
+        return JsonResponse({'account_id': account.id}, status=201)
+
+
+@method_decorator(csrf_exempt, name='dispatch')
```
Comment (misslame) [inline mesh/accounts/views.py:55]: Why make it csrf exempt

Comment (bluesimp1102) [reply mesh/accounts/views.py]: I removed it. The initial thought was having it there to having the protection but I didn't realize we won't have it if I put it there.

**Review by owwix — CHANGES_REQUESTED** [review]
remove use of csrf exempt decorators, the csrf middleware is temporarily disabled.

**Review by GuyWhoCode — APPROVED** [review]
Nice!

**Review by owwix — APPROVED** [review]

**Pull request merged**